### DISCUSSION TITLE
cargo-mutants-1.2.1

### DISCUSSION THREAD

**Discussion by sourcefrog** [sourcefrog/cargo-mutants#104] [Announcements]
- Converted most of the docs to a book available at <https://mutants.rs/>.

- Fixed: Correctly find submodules that don't use `mod.rs` naming, e.g. when descending from `src/foo.rs` to `src/foo/bar.rs`. Also handle module names that are raw identifiers using `r#`. (Thanks to @kpreid for the report.) 

<hr /><em>This discussion was created from the release <a href='https://github.com/sourcefrog/cargo-mutants/releases/tag/cargo-mutants-1.2.1'>cargo-mutants-1.2.1</a>.</em>

**Comment by JeftavanderHorst** [comment]
The [mutants.rs](https://mutants.rs/) website is a 404 for me

**Reply by sourcefrog** [reply]
Oh thanks for pointing that out. It was working before. I'll investigate. #105

**Reply by sourcefrog** [reply]
I think I fixed it by running the publishing workflow again.
